AssignGamesManual.py

```diff
@@ -1,28 +1,28 @@
-from Objects import *
+from Objects import Round, Game, GetPlayerByPairingNumber, GetCombinedPreference, GetPlayerWeightForGame
```

Comment:
het is een goed idee om expliciet je imports te doen zodat je niet per ongeluk andere zaken mee importeert (zoals hier bijv. logging).